apps/settings/js/modules/panel_cache.js

```diff
@@ -32,6 +32,7 @@ define(function(require) {
                        'style/simcard.css',
                        'style/updates.css',
                        'style/downloads.css',
+                       'style/improve_browser_os.css',
```

Comment:
I understand the loadtime performance issue, but all the other panel style sheets are lazily loaded. It seems inconsistent for this panel style sheet to be treated differently.